plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -115,7 +122,7 @@ export function AboutCard(props: AboutCardProps) {
       viewTechdocLink &&
       viewTechdocLink({
         namespace: entity.metadata.namespace || DEFAULT_NAMESPACE,
-        kind: entity.kind,
+        kind: toLowerMaybe(entity.kind, config),
         name: entity.metadata.name,
```

Comment:
I believe all parts of the entity ref needs to be processed by `toLowerMaybe()`?